2024/2024-ssl-black-white-hulls-rules.adoc

```diff
@@ -29,23 +29,27 @@ All teams are eligible and *are encouraged* to participate in this challenge.
 
 == Evaluation
 
-Every team has a 5-minute pitch to explain their solution for the challenge and the built design and 5 minutes for possible questions. During the presentation, the TC/OC will measure the time of the team solution.
+Every team has a 5-minute pitch to explain their solution for the challenge and the built design and 5 minutes for possible questions.
+
+After the presentation, the TC/OC will measure the time of the team solution. The TC/OC will decide on the venue how the measurement will occur, but in general it will be measured the time to change the hull of one robot, considering the vision pattern.
 
 At least one member of each team should be present during presentations since all teams will vote for the best solution.
 
 The following criteria will be evaluated.
 
-* Size efficiency
-* Durability
-* Ease of changing
-* Costs
-* Ease of building
+* *Size efficiency:* this criterion evaluates the efficiency of the logistics to transport the hulls.
+* *Durability:* this criterion evaluates the resistance of the solution to possible crashes and other possible damages during a game.
+* *Ease of changing:* this is a qualitative criterion to measure how the other teams feel about the time and ease of the solution proposed.
+* *Costs:* this criterion evaluates the costs of the solution, like material used or machining.
+* *Ease of building:* this criterion evaluates how easy the process is to build the solution proposed.
 
 === Scoring
 
-After the presentations, every team will receive a ballot where they will rate the solutions from 1 to 5 for each criterion.
+After the presentations, every team will receive a ballot where they will rate the solutions from 0 to 5 for each criterion.
+
+The teams are encouraged to use their solutions in the matches to prove their efficiency to the others. Besides, the teams that use their solution in a game earn 5 extra points and can receive 5 additional points if they switch the hulls of at least four robots during half-time.
```

Comment:
I don't know if we need  `of at least four robots during half-time`. 

If a team is playing with less than 4 robots, their focus is to fix the robots, instead of changing hull. However, there is no problem in keeping it :)

Reply:
Yes, I was trying to don't be unfair to teams of Div A since they have more robots than Div B teams. But, maybe the minimum amount is indeed not needed. 

Maybe instead of 5 points for the changes in half-time, the extra points may depend on the number of hulls changed, like 2 points per change. But, the referees would need to count it in the half time and report in the Score Sheet. Maybe it would be better for the referee to ask before the match if the team would perform the BW Challenge at half-time (if the team is participating in the challenge), and the points will only be considered if the team decides for it.

Reply:
Let's keep it simple now. Also, teams might be planning for four right now. For next year we can start thinking on extra points only if teams change the whole team :D